ui/raidboss/data/04-sb/eureka/eureka_hydatos.ts

```diff
@@ -112,19 +112,22 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'BA West Side',
       type: 'Ability',
-      netRegex: NetRegexes.ability({ id: '3956', source: 'Art', target: '[^:]+', capture: false }),
+      // The two sides are far enough apart that even though you see the autos
+      // from the opposite boss, you don't see the player target (which is blank).
+      // This considers you on the west side if Art autos somebody, and the name isn't blank.
+      netRegex: { id: '3956', source: 'Art', target: '[^:]+', capture: false },
       suppressSeconds: 1000,
       run: (data) => data.side = 'west',
     },
     {
       id: 'BA East Side',
       type: 'Ability',
-      netRegex: NetRegexes.ability({
+      netRegex: {
         id: '3957',
         source: 'Owain',
         target: '[^:]+',
```

Comment:
Also, I have no idea how this is working given that `[^:]+` is meant for parsed log lines and not network log lines, but I'll change that in another PR.

Reply:
There's no reason this wouldn't work. Assuming this targets players, a player's name can never have a `:` character anyways, so it's functionally identical to `target: '[^|]'`. Maybe that ability can target non-player actors and the colon check is actually important though? I have no clue, I've never even stepped into Eureka.

Reply:
Well, since this is building into a regular expression, I didn't want it to match a blank entry, e.g. `|playerid||` kinda thing, where `[^:]+` matched the final bar.